src/current/v22.2/advanced-changefeed-configuration.md

```diff
@@ -19,6 +19,39 @@ Some options for the `kafka_sink_config` and `webhook_sink_config` parameters ar
 - [Kafka sinks](changefeed-sinks.html#kafka-sink-configuration)
 - [Webhook sinks](changefeed-sinks.html#webhook-sink-configuration)
 
+### MuxRangefeed
+
+{{site.data.alerts.callout_info}}
+{% include feature-phases/preview.md %}
+{{site.data.alerts.end}}
+
+`MuxRangefeed` is a subsystem that improves the performance of rangefeeds with scale. Its functionality is enabled via the `changefeed.mux_rangefeed.enabled` [cluster setting]({% link {{ page.version.version }}/cluster-settings.md %}). We recommend large-scale changefeed workloads enable this cluster setting.
```

Comment:
I've added MuxRangefeed to v22.2, v23.1, v23.2 (preview in the earlier two). 

**Do we want to be more precise about what kind of users should be enabling this?** Whether we can size numbers, instead of "large-scale", or are there any other workload/cluster characteristics that would make it ideal for enabling Mux?

Reply:
All users in my opinion; but then we didn't make it on by default.
All users are welcome to try it -- this is an "early preview" feature that will be enabled by default in the 24.1 release?

Actually, no -- I'd keep this recommendation as is for 22.2 and 23.1; but
encourage people to enable it in 23.2 as the feature will be on by default in 24.1